pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -39,6 +40,12 @@ def on_trainer_init(
         # init folder paths for checkpoint + weights save callbacks
         self.trainer._default_root_dir = default_root_dir or os.getcwd()
         self.trainer._weights_save_path = weights_save_path or self.trainer._default_root_dir
+        if stochastic_weight_avg:
+            rank_zero_deprecation(
+                "Setting `Trainer(stochastic_weight_avg=True)` is deprecated in v1.5 and will be removed in v1.7."
+                " Please pass `pytorch_lightning.callbacks.stochastic_weight_avg.StochasticWeightAveraging`"
+                " directly to the Trainer's ``callbacks`` argument instead."
```

Comment:
```suggestion
                "Setting `Trainer(stochastic_weight_avg=True)` is deprecated in v1.5 and will be removed in v1.7."
                " Please pass `pytorch_lightning.callbacks.stochastic_weight_avg.StochasticWeightAveraging`"
                " directly to the Trainer's `callbacks` argument instead."
```
for the python message